packages/typespec-test/test/healthInsights_radiologyinsights/tspconfig.yaml

```diff
@@ -1,12 +1,11 @@
-emit: [
-    "@azure-tools/typespec-ts",
-  ]
+emit: ["@azure-tools/typespec-ts"]
 options:
   "@azure-tools/typespec-ts":
     generateMetadata: true
     generateTest: true
     generateSample: true
     azureSdkForJs: false
+    isModularLibrary: true
```

Comment:
Since healthinsight has very complicated nested model definition, would like to keep this option in our smoke test. We still have another healthinsight case for rlc.